docs/units-and-global-variables.rst

```diff
@@ -311,14 +311,17 @@ For more information, see the section on :ref:`address`.
     semantics than ``delegatecall``.
 
 
-.. index:: this, selfdestruct
+.. index:: this, selfdestruct, super
 
 Contract Related
 ----------------
 
 ``this`` (current contract's type)
-    the current contract, explicitly convertible to :ref:`address`
+    The current contract, explicitly convertible to :ref:`address`
 
+``super``
+    The contract one level higher in the inheritance hierarchy
```

Comment:
```suggestion
    The parent contract which the current contract inherits from
```

Reply:
I'm not sure if that's really better. Both seem to imply that there's just one contract while `super` is a really weird thing that can refer to inherited stuff from multiple contracts from multiple levels.

Reply:
I changed it to "a contract", which is better, but we should find better wording at some point.